templates/default/rt/rtfilter.html

```diff
@@ -76,8 +76,10 @@
             </label>
             <label class="nobr">
                 {trans("Verifier:")}
-                <input type="hidden" name="vids" value="all">
```

Comment:
Wiesz po co był ten hidden? Po to, żeby jak nie wybierzesz żadnej wartości dla `vids` to przyjęła wartość `all`.
Rozumiem, że nie obawiasz się przypadku w którym żaden checkbox  z multiselect nie zostanie wybrany?

Reply:
Na backendzie jest każdy przypadek https://github.com/lmsgit/lms/blob/73b50b8a93d522ccf913ad43200b20ba70abc595/lib/LMSManagers/LMSHelpdeskManager.php#L223-L233

Czy ten `all` ma sens? Nie wszędzie jest w filtrach to...

Reply:
@chilek to jak? zostawić ten input czy może zostać usunięty? 
Czy jeśli zostawiamy go to: dodać ten parametr na backend i do dokumentacji funkcji?

Reply:
Gdy to usuniesz to brak vids nie pozwoli odróżnić sytuacji w której w ogóle nie ma wyboru, bo formularz jeszcze się nie ładowało od sytuacji, że ma wszystkie pokazywać.

Reply:
ok - przywróciłem,